fw/t/unit/test_http_parser_common.h

```diff
@@ -669,11 +663,9 @@ do {									\
 #define FOR_RESP(str)	__FOR_RESP(str, 0, CHUNK_ON)
 
 #define EXPECT_BLOCK_RESP(str)						\
-do {									\
 	TEST_LOG("=== response: [%s]\n", str);				\
 	test_case_parse_prepare_http(str);				\
-	TRY_PARSE_EXPECT_BLOCK(FUZZ_RESP, CHUNK_ON);			\
-} while (0)
+	TRY_PARSE_EXPECT_BLOCK(FUZZ_RESP, CHUNK_ON)
```

Comment:
I believe there is no reason to remove `do { } while (0)` statements.

Reply:
No, if `do { } while (0)` remains, then the checks from the check block will always be performed only for the last chunk size. More precisely, these checks will always work, regardless of the result of running TRY_PARSE_EXPECT_BLOCK.